### PR TITLE
[MGDAPI-1349] adding prow check

### PR DESCRIPTION
**Pull request by maksymvavilov**
## What 
added prow check for files generated by the kubebuiler 

## Why 
to check that files were not edited manually 
https://issues.redhat.com/browse/MGDAPI-1349

### CODE REVIEW

**Comment by makslion** [comment]
@laurafitzgerald am I doing huge mistakes here?

**Comment by laurafitzgerald** [comment]
@makslion there is another step to do to generate the job config. Run `make jobs` against this pr which should add this new prow target to the job definitions. It's important that your branch is up to date with master otherwise there could be other files generated which don't relate to your change. Let me know how it goes. 👍

**Comment by laurafitzgerald** [comment]
fyi https://github.com/openshift/release/pull/16538 related pr which we can add the check to also. 👍

File: ci-operator/config/integr8ly/integreatly-operator/integr8ly-integreatly-operator-master.yaml [hunk 1]
```diff
@@ -68,6 +68,10 @@ tests:
   commands: make test-cases/lint
   container:
     from: src
+- as: kubebuilder
+commands: make kubebuilder/check
+container:
+  from: src
```
Comment (laurafitzgerald) [inline ci-operator/config/integr8ly/integreatly-operator/integr8ly-integreatly-operator-master.yaml:74]:
```suggestion
- as: kubebuilder
  commands: make kubebuilder/check
  container:
    from: src
```

**Comment by laurafitzgerald** [comment]
pr https://github.com/integr8ly/integreatly-operator/pull/1638 needs to merge before we can merge this one.

**Comment by laurafitzgerald** [comment]
@makslion please retest the failures once the pr mentioned above is merged. it's expected that they will fail until the prow make target is on master. 👍

**Comment by laurafitzgerald** [comment]
@makslion it's failing on `make: *** No rule to make target `kubebuilder/check'.  Stop.` so it's not finding the target. I think this usually happens on new prow jobs. I'm going to approve it so it can merge and let's check on a pr to the integreatly-operator repo if it works. 👍 
/lgtm

**Pull request merged**